docs/roles/browsers.rst

```diff
@@ -0,0 +1,49 @@
+*************
+Browser Roles
+*************
+
+Browser roles are available for frontend testing. They all depend on
+the role xvfb, which is a headless X server.
+
+
+Firefox
+=======
+
+Install Firefox with Geckodriver to be used with selenium.
+
+
+Parameters
+----------
+
+* **firefox\_version** : The version of Firefox to be installed, defaults to
```

Comment:
Please add a link to https://ftp.mozilla.org/pub/firefox/releases/ for available versions.